include/glow/Importer/ONNX.h

```diff
@@ -89,6 +89,10 @@ class ONNXModelLoader
   size_t opsetVersion_;
 
 public:
+  void CheckIO(onnx::GraphProto &net,
```

Comment:
Doxygen comment please.

Reply:
@jfix71 It only checks the input shape, so checkInput() would be more accurate.

Reply:
Renamed : checkInputs()